book/01-introduction/sections/help.asc

```diff
@@ -10,18 +10,18 @@ $ git <verb> --help
 $ man git-<verb>
 ----
 
-Beispielweise erhalten Sie die Hilfeseite für den `git config` Befehl folgendermaßen:(((Git Befehle, help)))
+Beispielweise erhalten Sie die Hilfeseite für den `git config`-Befehl folgendermaßen:(((Git Befehle, help)))
 
 [source,console]
 ----
 $ git help config
 ----
 
 Diese Befehle sind nützlich, weil Sie sich die Hilfe jederzeit anzeigen lassen können, auch wenn Sie einmal offline sind.
-Wenn die Hilfeseiten und dieses Buch nicht ausreichen und Sie persönliche Hilfe brauchen, können Sie den `#git` oder `#github` Kanal auf dem Freenode IRC Server probieren, der unter https://freenode.net zu finden ist.
+Wenn die Hilfeseiten und dieses Buch nicht ausreichen und Sie persönliche Hilfe brauchen, können Sie den `#git`- oder `#github`-Kanal auf dem Freenode-IRC-Server probieren, der unter https://freenode.net zu finden ist.
```

Comment:
Deine Korrektur ist grammatikalisch richtig, aber hast du dir das mal optisch angeschaut?
Die zusätzlichen Trennstriche "zerreißen" den Lesefluss.

Reply:
Für mich ist das so in Ordnung. Falls damit wirklich der Lesefluss zerrissen werden sollte, wie wäre es mit der Alternative

```suggestion
Wenn die Hilfeseiten und dieses Buch nicht ausreichen und Sie persönliche Hilfe brauchen, können Sie einen der Kanäle `#git` oder `#github` auf dem Freenode-IRC-Server probieren, der unter https://freenode.net zu finden ist.
```